### PR TITLE
Write explanation for why we combined sprint 2 and 3

### PR DESCRIPTION
**Pull request by chhoumann**

### CODE REVIEW

File: sections/Chapter 5 Sprint 3/Sprint3.tex [hunk 1]
```diff
@@ -1,21 +1,9 @@
 \chapter{Sprint 3}
+We have chosen to combine our descriptions of the two previous sprints into one sprint. Sprint 2 was supposed to be the sprint where the search engine was fixed, but ended up being swallowed by planning and bureaucracy. For this reason, it also took sprint 3 to fully implement the search engine. Because of our choice to combine the two sprints, any reference to sprint 3 should be interpreted as referring to the combined sprint.
```
Comment (RasmusHoeyerHansen) [inline sections/Chapter 5 Sprint 3/Sprint3.tex:2]: Maybe "swallowed by planning and collaboration efforts between the groups knox."

Kunne være rart at vide hvor langt sprint 3 så egentlig var.

**Review by RasmusHoeyerHansen — CHANGES_REQUESTED** [review]

File: sections/Chapter 5 Sprint 3/Sprint3.tex [hunk 2]
```diff
@@ -1,5 +1,6 @@
 \chapter{Sprint 3}
-We have chosen to combine our descriptions of the two previous sprints into one sprint. Sprint 2 was supposed to be the sprint where the search engine was fixed, but ended up being swallowed by planning and bureaucracy. For this reason, it also took sprint 3 to fully implement the search engine. Because of our choice to combine the two sprints, any reference to sprint 3 should be interpreted as referring to the combined sprint.
+We have chosen to combine our descriptions of the two previous sprints into one sprint. Sprint 2 was supposed to be the sprint where the search engine was fixed, but ended up being swallowed by planning and collaboration efforts between the \knox{} groups. For this reason, it also took sprint 3 to fully implement the search engine. Because of our choice to combine the two sprints, any reference to sprint 3 should be interpreted as referring to the combined sprint.
+All in all the sprint lasted from 21st of September to 4th of October.
```
Comment (chhoumann) [inline sections/Chapter 5 Sprint 3/Sprint3.tex:3]: Comma

Comment (chhoumann) [reply sections/Chapter 5 Sprint 3/Sprint3.tex:3]:
```suggestion
All in all the, sprint lasted from 21st of September to 4th of October. 
```

**Review by chhoumann — COMMENTED** [review]
GJ

**Pull request merged**